adapters/relevantdigital/relevantdigital.go

```diff
@@ -0,0 +1,334 @@
+package relevantdigital
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+	"net/http"
+	"strings"
+	"text/template"
+
+	"github.com/buger/jsonparser"
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/macros"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+	jsonpatch "gopkg.in/evanphx/json-patch.v4"
+)
+
+type adapter struct {
+	endpoint *template.Template
+	name     string
+}
+
+const relevant_domain = ".relevant-digital.com"
+const default_timeout = 1000
+const default_bufffer_ms = 250
+
+type PrebidExt struct {
+	StoredRequest struct {
+		Id string `json:"id"`
+	} `json:"storedrequest"`
+	Debug bool `json:"debug"`
+}
+
+type RelevantExt struct {
+	Relevant struct {
+		Count       int    `json:"count"`
+		AdapterType string `json:"adapterType"`
+	} `json:"relevant"`
+	Prebid PrebidExt `json:"prebid"`
+}
```

Comment:
scope of `PrebidExt` and `RelevantExt` is limited to relevantdigital package

therefore could make these struct private

```
type prebidExt struct {
	StoredRequest struct {
		Id string `json:"id"`
	} `json:"storedrequest"`
	Debug bool `json:"debug"`
}

type relevantExt struct {
	Relevant struct {
		Count       int    `json:"count"`
		AdapterType string `json:"adapterType"`
	} `json:"relevant"`
	Prebid PrebidExt `json:"prebid"`
}

```

Reply:
Thanks, fixed :)